content/articles/activity-tracking.markdown

```diff
@@ -0,0 +1,49 @@
+---
+title: Activity Tracking
+excerpt: How to manage the users on a DNSimple account.
+categories:
+- Account
+---
+
+# Managing account users
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+<note>
+Personal plan subscribers have access to all events that occurred during the last 30 days. A [Professional and Business subscription](https://dnsimple.com/pricing) is required to view the full history.
```

Comment:
We generally link the /new-plans#newer-plans-some in this case.
See e.g. https://github.com/dnsimple/dnsimple-support/blob/master/content/articles/regional-records.markdown

I'd also rephrase

> is required to view the full history.

as it sees mandatory. I'd use something like:

> Audit history is available only for new plans. Full audit history is available for Professional and Business plans, Personal plans audit history is limited to 30 days.